elasticapm/contrib/serverless/aws.py

```diff
@@ -246,6 +256,18 @@ def set_metadata_and_context(self, coldstart: bool) -> None:
             cloud_context["origin"]["service"] = {"name": "api gateway"}
             cloud_context["origin"]["account"] = {"id": self.event["requestContext"]["accountId"]}
             cloud_context["origin"]["provider"] = "aws"
+        elif self.source == "elb":
+            elb_target_group_arn = self.event["requestContext"]["elb"]["targetGroupArn"]
+            faas["trigger"]["type"] = "http"
+            faas["trigger"]["request_id"] = self.event["headers"]["x-amzn-trace-id"]
+            service_context["origin"] = {"name": elb_target_group_arn.split(":")[5]}
+            service_context["origin"]["id"] = elb_target_group_arn
+            service_context["origin"]["version"] = self.event.get("version", "1.0")
+            cloud_context["origin"] = {}
```

Comment:
```suggestion
            service_context["origin"]["id"] = elb_target_group_arn
            cloud_context["origin"] = {}
```

Let's just omit the origin.version field here. Since we're not using API Gateway I don't think there's anything useful we can put in that field for ELB.